defaults/firefox_ios.toml

```
@@ -47,6 +47,17 @@ bootstrap_mean = {}
 
 ##
 
+[metrics.serp_ad_clicks]
+friendly_name = "Ad Clicks"
+description = "Number of ad clicks on a search engine results page."
+select_expression = "{{agg_sum('ad_click')}}"
+data_source = "mobile_search_clients_engines_sources_daily"
+
+[metrics.serp_ad_clicks.statistics]
+deciles = {}
+
```

It looks like in the fenix config is also computes the bootstrap mean. Not sure if we'd want that here as well?
```suggestion
bootstrap_mean = {}
```